### PR TITLE
Explicitly support Go 1.6

### PR DESCRIPTION
**Pull request by mvdan**
CI: remove `go get -d` lines.

We vendor all deps, so this is unnecessary.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+  - 1.6.x
   - 1.7.x
 
 services:
   - redis-server
   - mongodb
 
 install:
-  - go get -d -v ./...
-  - go get -d -v -tags coprocess ./...
```
Comment (buger) [inline .travis.yml]: What is the reason of removing this lines?

Comment (mvdan) [inline .travis.yml]: See discussion on slack. They do nothing, as they would only download but everything is already on disk since all deps are vendored.

Comment (mvdan) [inline .travis.yml]: (also explained in the commit, let me know if you'd like my commits to be more verbose)

**Comment by buger** [comment]
👍

**Pull request merged**